recipes/example/meta.yaml

```diff
@@ -33,7 +33,7 @@ build:
   script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
-  build:
```

Comment:
Don't edit the example

Reply:
Oops, very sorry about that. I was using bulk find and edits.

Reply:
Done